src/syscheckd/db/CMakeLists.txt

```diff
@@ -78,7 +78,9 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
                            ${CMAKE_SOURCE_DIR}/src/file.cpp
                            ${CMAKE_SOURCE_DIR}/src/fimDB.cpp
                            # ${CMAKE_SOURCE_DIR}/src/registry.cpp
```

Comment:
Not anymore because the registry sync now works with transactions. However the current state of the registry interface will be kept in case of real-time mode implementation for registry monitorization.